src/integrators/integrator_utils.jl

```diff
@@ -253,7 +253,7 @@ function stepsize_controller!(integrator,alg::PredictiveControllerAlgs)
     q = inv(integrator.opts.qmax)
   else
     gamma = integrator.opts.gamma
-    if alg isa RKC
+    if alg isa Union{RKC,IRKC}
```

Comment:
`<:`, not `isa`

Reply:
@ChrisRackauckas 
getting this error when changing this
```
ERROR: TypeError: in <:, expected Type, got 
IRKC{0,true,LinSolveFactorize{typeof(LinearAlgebra.lu!)},
NLNewton{true,OrdinaryDiffEq.NLNewtonCache{Nothing,Nothing,Nothing,
Nothing,Nothing,Nothing}},DataType,Nothing,Nothing,Float64,Int64,:Predictive}
```